elasticsearch/config/config_secure.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Wazuh App Copyright (C) 2019 Wazuh Inc. (License GPLv2)
+
+elastic_config_file="/usr/share/elasticsearch/config/elasticsearch.yml"
+
+
+##############################################################################
+# Setup bootstrap password to chagne all Elastic Stack passwords.
+# Set xpack.security.enabled to true. In Elastic 7 must add ssl options
+##############################################################################
+
+if [[ $SETUP_PASSWORDS == "yes" ]]; then
+
+  echo "Creating certificate."
+
+  pushd /usr/share/elasticsearch/config/
+
+  echo "Setting configuration options."
+
+  echo "
+instances:
+- name: \"elasticsearch\"
+  dns: 
+    - $CERT_DNS
+" > instances.yml
+
+  unzip elastic-CA.zip
+  /usr/share/elasticsearch/bin/elasticsearch-certutil cert --pem -in instances.yml --out certs.zip --ca-cert server.CA-signed.pem --ca-key server.CA.key  --ca-pass $CA_PASS
+  unzip certs.zip
+
+  popd
+
+  chown elasticsearch: /usr/share/elasticsearch/config/server.CA-signed.pem
+  chown -R elasticsearch: /usr/share/elasticsearch/config/elasticsearch
+  chmod 774 /usr/share/elasticsearch/config/server.CA-signed.pem
```

Comment:
Permissions should be 440 for files.

Reply:
Changed, we will test with 440 permissions.